README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Github Actions Status](https://github.com/jupyterlab/mimerenderer-cookiecutter-ts/workflows/CI/badge.svg)
 
-A [cookiecutter](https://github.com/audreyr/cookiecutter) template for creating
+A [cookiecutter](https://github.com/cookiecutter/cookiecutter) template for creating
 a JupyterLab MIME renderer extension in TypeScript. (See also
 [mimerender-cookiecutter](https://github.com/jupyterlab/mimerender-cookiecutter)
 for a MIME renderer extension in CommonJS.)
```

Comment:
Should the link to this archived repository be kept ?

Reply:
Thanks for catching that, the new link sounds good.